src/act_wiz.cpp

```diff
@@ -1278,23 +1272,14 @@ void do_mstat(CHAR_DATA *ch, const char *argument) {
         return;
     }
 
-    bug_snprintf(buf, sizeof(buf), "Name: %s     Clan: %s     Rank: %s.\n\r", victim->name,
-                 victim->clan() ? victim->clan()->name : "(none)",
-                 victim->pc_clan() ? victim->pc_clan()->level_name() : "(none)");
-    send_to_char(buf, ch);
+    ch->send_to("Name: {}     Clan: {}     Rank: {}.\n\r"_format(
+        victim->name, victim->clan() ? victim->clan()->name : "(none)",
+        victim->pc_clan() ? victim->pc_clan()->level_name() : "(none)"));
 
-    do {
-        if (snprintf(buf, sizeof(buf), "Vnum: %d  Format: %s  Race: %s  Sex: %s  Room: %d\n\r",
-                     victim->is_npc() ? victim->pIndexData->vnum : 0, victim->is_npc() ? ".are" : "pc",
-                     race_table[victim->race].name, victim->sex == 1 ? "male" : victim->sex == 2 ? "female" : "neutral",
-                     victim->in_room == nullptr ? 0 : victim->in_room->vnum)
-            < 0)
-            bug("Buffer too small at "
-                "_file_name_"
-                ":"
-                "1296"
-                " - message was truncated");
-    } while (0);
+    bug_snprintf(buf, sizeof(buf), "Vnum: %d  Format: %s  Race: %s  Sex: %s  Room: %d\n\r",
```

Comment:
eventually bug_snprintf() itself can be ditched I suspect

Reply:
Absolutely!